.github/actions/action-connect-ovpn/action.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Connect-VPN-action'
+description: 'Connect VPN action'
+branding:
+  icon: 'shield'
+  color: 'orange'
+inputs:
+  SECRET:
+    description: 'Username and password for access vpn'
+    required: false
+    default: ''
+  TLS_KEY:
+    description: 'User key for access vpn'
+    required: false
+    default: ''
+  PING_URL:
+    description: 'For check success or fail'
+    required: true
+    default: '127.0.0.1'
+  FILE_OVPN:
+    description: 'Location file open vpn'
+    required: true
+    default: './config.ovpn'
+outputs:
+  STATUS:
+    description: 'Status for check connect vpn'
+    value: ${{ steps.vpn_status.outputs.vpn-status }}
+runs:
+  using: "composite"
+  steps:
+    - name: Install OpenVPN
+      run: |
+        sudo apt-get update
+        sudo apt-get install openvpn
+        sudo apt-get install openvpn-systemd-resolved
+      shell: bash
+
+    - name: Connect VPN
+      env:
+        TLS_KEY: ${{ inputs.TLS_KEY }}
```

Comment:
validate inputs and env vars are set

Reply:
We have this input defined in the Environment Secrets section.

Reply:
All these issues are not documented and we are in the process of reaching out to Mr. Josiah Siegel and ask for his assistance in figuring out the different environment and secrets we need to setup in this project.